app-config.yaml

```diff
@@ -47,10 +47,22 @@ integrations:
     #   token: ${GHE_TOKEN}
 
 proxy:
-  '/harness':
+  '/harness/prod':
     target: 'https://app.harness.io/'
     headers:
       'x-api-key': ${HARNESS_API_KEY}
+  '/harness/qa':
+    target: 'https://qa.harness.io/'
+    headers:
+      'x-api-key': ${HARNESS_API_KEY}
```

Comment:
If you could rename the Environment Variables Names, would be better to inject them later